src/components/Goodlist/GoodsList.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+  List,
+  ListItem,
+  ListItemText,
+} from '@mui/material';
+import { Good } from '../../types/Good';
+
+type Props = {
+  goods: Good[]
+};
+
+export const GoodsList: React.FC<Props> = React.memo(
+  ({ goods }) => (
+    <List
+      sx={{
+        width: '100%',
+        position: 'relative',
+        overflow: 'auto',
+        maxHeight: 250,
+        '& ul': { padding: 0 },
+      }}
+      subheader={<li />}
+    >
+      <ul>
```

Comment:
I think you can get rid of this `ul` when using `List` component to avoid unnecessary nesting

![image](https://user-images.githubusercontent.com/48877776/227597351-edbc3575-8177-40a4-a1c4-86e8d5b5d61e.png)

Reply:
you're right, fixed)